sdk/all/pr.client.yml

```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT THIS FILE
+# This file is be generated automatically and any changes will be lost.
```

Comment:
Just noticed the typo, need to remove "be"

Reply:
Fixed.